Description: drop test not compatible with coffee 2
Author: Yadd <yadd@debian.org>
Bug-Debian: https://bugs.debian.org/1013620
Forwarded: not-needed
Last-Update: 2022-06-24

--- a/test/parser.test.coffee
+++ b/test/parser.test.coffee
@@ -497,17 +497,6 @@
       equ err, null
       test.finish()
 
-  'test construction with new and without': (test) ->
-    demo = '<xml><foo>Bar</foo></xml>'
-    withNew = new xml2js.Parser
-    withNew.parseString demo, (err, resWithNew) ->
-      equ err, null
-      withoutNew = xml2js.Parser()
-      withoutNew.parseString demo, (err, resWithoutNew) ->
-        equ err, null
-        assert.deepEqual resWithNew, resWithoutNew
-        test.finish()
-
   'test not closed but well formed xml': (test) ->
     xml = "<test>"
     xml2js.parseString xml, (err, parsed) ->
